packages/workbox-core/_default.mjs

```diff
@@ -49,12 +49,19 @@ class WorkboxCore {
     if (process.env.NODE_ENV !== 'production') {
       const padding = '   ';
       logger.groupCollapsed('Welcome to Workbox!');
+      if (self.location.hostname === 'localhost') {
+        logger.unprefixed.log(`The development build with extra logging is ` +
```

Comment:
This can also happen on localhost because of the config options.

I'm personally leaning towards having a shorter more direct - You are currently using a development build. By default this will switch to prod builds when not on localhost. You can force this with `workbox.setConfig({debug: true|false|)`.